### ISSUE TITLE
[Introspection] Better `introspect` success message

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#1554]
A successful `introspect` run currently looks like this:

```
λ npx prisma2@alpha introspect
Introspecting …
Done with introspection in 1.42s
Wrote schema.prisma
```

We should probably try to make this a bit nicer.

1. The text itself can be a lot more descriptive ("datamodel was added to your `schema.prisma`") and for example offer next steps ("Now use `prisma2 generate` to create a Prisma Client for your database").
2. We could also output some statistics about the datamodel that was added: x models, y relations, z indexes (Actually only the "model" part might be reasonably simple, but maybe this is a good place to go a bit overbord and have fun)

**Comment by sorenbs** [comment]
I would love to have statistics as suggested in 2.

We should also think about how to display failures / guard rails. Something like: 2 models and 7 relations were disabled because we couldn't support them.

I don't think we should print "next steps" as suggested in 1. It's too much clutter.

I think it would be nice to add a green checkmark ala firebase:

![image](https://user-images.githubusercontent.com/281337/74102932-38bf2180-4b48-11ea-85a5-0414aefda606.png)

Whatever we decide, the result should be a pr to the [CLI spec](https://github.com/prisma/specs/tree/master/cli).

**Comment by janpio** [comment]
> We should also think about how to display failures / guard rails. Something like: 2 models and 7 relations were disabled because we couldn't support them.

Yes, we will flesh that out separately though when we actually implement the guardrail handling - for now we just assume everything gets written to the schema.

**Comment by janpio** [comment]
Related: https://github.com/prisma/prisma2/issues/1574

**Comment by Jolg42** [comment]
Here is the current output for Discourse with latest alpha 2.0.0-alpha.1182 (same with beta 4)

`npx @prisma/cli@alpha introspect`

<img width="530" alt="Screen Shot 2020-05-04 at 11 42 56" src="https://user-images.githubusercontent.com/1328733/80954231-48739c80-8dfd-11ea-89c7-346da4c43e54.png">

**Comment by do4gr** [comment]
I think the only valuable statistic we could add to the output is the number of generated models. This should in most cases map cleanly to the number of tables (except for legacy Prisma generated abs that contain Jointables, Listtables and Migration tables).
 
We could also add number of enums, but since on MySQL enums cannot be reused and we generate one per enum column this might be confusing.

Fields could be confusing since it would not be expected to map cleanly to db columns (since we add relation fields and backrelationfields).

Number of relations is something we could add as well, but again this does not have to cleanly map to number of foreign keys due to duplicates and/or implicit join relations. 

Number of Indexes  could also be confusing, since indexes can be represented in different ways in the datamodel (@@index/@unique/@@unique/@id/@@id).

All of this information could be retrieved from the dmmf (apart from indexes). This would mean that we do not need to change the interfaces of either Introspection Engine or Query Engine and so that would be my preferred approach.

**Comment by janpio** [comment]
Then I suggest this iteration:

> ✔ Introspected 17 models and wrote them into `prisma/schema.prisma` in **1.36s**

It changes the checkmark to a green one as suggested by @sorenbs, adds the number of models as suggested by @do4gr and outputs the actual path of `schema.prisma`.

If we agree on that I would persist it in https://github.com/prisma/specs/tree/master/cli#introspect so it can be implemented.

**Comment by janpio** [comment]
Persisted in https://github.com/prisma/specs/pull/471, will hand off to implementation after this is merged.

**Comment by Jolg42** [comment]
@janpio It was [fixed by @do4gr](https://github.com/prisma/prisma-engines/commit/86cf27779951480f87f6bbdb15a3722c5756a8e4) and shipped in 2.0.0-alpha.1218

Example
<img width="535" alt="Screen Shot 2020-05-18 at 12 06 02" src="https://user-images.githubusercontent.com/1328733/82201152-37985000-9900-11ea-84a4-bd1380d8e3fb.png">

with --print
<img width="526" alt="Screen Shot 2020-05-18 at 12 05 37" src="https://user-images.githubusercontent.com/1328733/82201159-39621380-9900-11ea-845e-57d0c20e1bd8.png">
